core-it-suite/src/test/java/org/apache/maven/it/MavenITmng6772NestedImportScopeRepositoryOverride.java

```diff
@@ -0,0 +1,95 @@
+package org.apache.maven.it;
+
+import java.io.File;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.maven.it.util.ResourceExtractor;
+
+/**
+ * This is a test set for <a href="https://issues.apache.org/jira/browse/MNG-6772">MNG-6772</a>:
+ *
+ * The test POM references an import scope POM, which also has a dependency on an import scope POM.
+ *
+ * Both import POMs can only be found in the repository defined in the test POM. 
+ * It has a parent POM that defines the same repository with a different location.
+ * The test confirms that the dominant repository definition (child) wins while resolving the import POMs.
+ *
+ */
+public class MavenITmng6772NestedImportScopeRepositoryOverride
+    extends AbstractMavenIntegrationTestCase
+{
+
+    public MavenITmng6772NestedImportScopeRepositoryOverride()
+    {
+        super( "(3.6.2,)" );
```

Comment:
If this is a regression, please change it to `super( "(,3.6.2),(3.6.2,)" );` so we can verify it with older versions of Maven.

Reply:
It is not a regression that I am aware of. The tests fail on all released 3.5.x and 3.6.x versions. I can't guarantee that there isn't a 3.3.x version or older where this may work, but seems like a lot of effort to figure out the answer.